core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosWithReservedKeywordIntegrationTest.java

```diff
@@ -0,0 +1,63 @@
+package com.scalar.db.storage.cosmos;
+
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.StorageWithReservedKeywordIntegrationTestBase;
+import java.util.Map;
+import java.util.Optional;
+
+public class CosmosWithReservedKeywordIntegrationTest
```

Comment:
I don't fully get this but this class seems not referenced at all. Is it OK?

Reply:
Seems the IDE still loading? There is no warning about that in my IDE. The CI running included this class already I think